web/dataselectie/datasets/brk/views.py

```diff
@@ -289,8 +289,12 @@ def load_from_elastic(self) -> dict:
         # the returned rows must be restricted by the size and page paramaters
         # not before.
         param_size = query['size']
-        offset = (int(self.request_parameters.get('page', 1)) - 1) * param_size
-        param_size = param_size * int(self.request_parameters.get('page', 1)) if offset > 0 else param_size
+        try:
+            param_page = int(self.request_parameters.get('page', 1))
+        except ValueError:
```

Comment:
Should this go unnoticed? Maybe inform the user about wrong input and return a 400 (bad request)?

Reply:
Check :+1: uiteindelijk helemaal verwijderd door toepassing collapse